system/vault/resources/httpproxy.yaml

```diff
@@ -3,22 +3,20 @@ kind: HTTPProxy
 metadata:
   name: vault
   namespace: vault
-  annotations:
-    cert-manager.io/cluster-issuer: letsencrypt-prod
 spec:
   virtualhost:
-    fqdn: vault.mgmt.cedille.club
+    fqdn: vault.mgmtv2.cedille.club
     tls:
       secretName: vault-tls
   routes:
-  - conditions:
-    - header:
-        contains: application/grpc
-        name: Content-Type
-    services:
-    - name: vault-ui
-      port: 8200
-      protocol: h2c
-  - services:
-    - name: vault-ui
-      port: 8200
+    - conditions:
+        - header:
+            contains: application/grpc
+            name: Content-Type
+      services:
+        - name: vault-ui
+          port: 8200
+          protocol: h2c
+    - services:
+        - name: vault-ui
+          port: 8200
```

Comment:
we should use the ingressClass contour-internal so that its only available internally and well have the vpn to give us access